test.csv

```diff
@@ -0,0 +1,3 @@
+Temperature,SOC

```

Comment:
There are 2 test.csv files, can you clarify why this is file exists.

Reply:
I have removed the test.csv. This was a mistake and i forgot to delete it.  Only the test.csv inside "Test_CSVfilePath" is used to test the file path and file existance.